main/options/options.cc

```diff
@@ -351,6 +351,11 @@ cxxopts::Options buildOptions() {
         "matchs. Matches must be against whole folder and file names, so `foo` matches `/foo/bar.rb` and "
         "`/bar/foo/baz.rb` but not `/foo.rb` or `/foo2/bar.rb`.",
         cxxopts::value<vector<string>>(), "string");
+    options.add_options("advanced")(
+        "lsp-dirs-not-on-client",
```

Comment:
I'm open to changing this CLI argument name. I couldn't think of a decent alternative.

Reply:
I don't know that I have a slam-dunk name, but maybe `directories-missing-from-client` or something like that? I feel like `lsp-dirs-not-on-client` makes it sound a bit like a flag instead of an argument that expects values.